codespell_lib/data/dictionary.txt

```diff
@@ -6365,6 +6588,7 @@ axissymmetric->axisymmetric
 axix->axis
 axixsymmetric->axisymmetric
 axpressed->expressed
+aymore->anymore
```

Comment:
For British English, perhaps:
```
aymore->anymore, any more,
```

Reply:
I will keep it as one word. Double typos like `aymore->any more` are not as useful.